### PR TITLE
[SYCL] Test for get_backend() methods

### PR DESCRIPTION
**Pull request by maximdimakov**
Test for the new get_backend() methods from https://github.com/intel/llvm/pull/3236
Signed-off-by: mdimakov <maxim.dimakov@intel.com>

### CODE REVIEW

File: SYCL/Basic/get_backend.cpp [hunk 1]
```diff
@@ -13,6 +13,11 @@
 
 using namespace cl::sycl;
 
+class DUMMY {
+public:
+  void operator()(item<1>){};
+};
+
```
Comment (s-kanaev) [inline SYCL/Basic/get_backend.cpp]: Is this class needed?

Comment (maximdimakov) [reply SYCL/Basic/get_backend.cpp]: I think this class is unnecessary now

Comment (maximdimakov) [reply SYCL/Basic/get_backend.cpp]: Removed

**Comment by maximdimakov** [comment]
@s-kanaev  could you please review the test?

**Review by s-kanaev — APPROVED** [review]

**Pull request merged**